### ISSUE TITLE
can't debug as usual

### ISSUE THREAD

**Issue by guizi-code** [spyder-ide/spyder#18422]
### What steps will reproduce the problem?

 debugger.run("runfile(%r, args=%r, wdir=%r, current_namespace=%r)"
 runfile() got an unexpected keyword argument 'current_namespace'

### Traceback
```python-traceback
  File "D:\Anaconda3\lib\site-packages\qtconsole\base_frontend_mixin.py", line 138, in _dispatch
    handler(msg)
  File "D:\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole\widgets\debugging.py", line 338, in _handle_input_request
    if self._hidden:
AttributeError: 'ShellWidget' object has no attribute '_hidden'
```

## Versions

* Spyder version: 4.1.4 
* Python version: 3.8.3
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  4.0.0 (OK)
cloudpickle >=0.5.0            :  2.0.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  8.3.0 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  6.4.4 (OK)
numpydoc >=0.6.0               :  1.2 (OK)
paramiko >=2.4.0               :  2.8.1 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.8.0 (OK)
pygments >=2.0                 :  2.11.2 (OK)
pylint >=1.0                   :  2.12.2 (OK)
pyls >=0.34.0;<1.0.0           :  0.35.1 (OK)
qdarkstyle >=2.8               :  3.0.2 (OK)
qtawesome >=0.5.7              :  1.0.3 (OK)
qtconsole >=4.6.0              :  5.3.0 (OK)
qtpy >=1.5.0                   :  2.0.1 (OK)
rtree >=0.8.3                  :  0.9.7 (OK)
sphinx >=0.6.6                 :  4.4.0 (OK)
spyder_kernels >=1.9.2;<1.10.0 :  1.9.4 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  22.3.0 (OK)

# Optional:
cython >=0.21                  :  0.29.28 (OK)
matplotlib >=2.0.0             :  3.5.1 (OK)
numpy >=1.7                    :  1.21.5 (OK)
pandas >=0.13.1                :  1.2.4 (OK)
scipy >=0.17.0                 :  1.7.3 (OK)
sympy >=0.7.3                  :  1.10.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @guizi-code, thanks for reporting. This error is already fixed in our latest version (5.3.1).

Since it's still not available in Anaconda, you have two options:

1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:

        conda create -n spyder-cf -c conda-forge spyder
        conda activate spyder-cf
        spyder

**Comment by guizi-code** [comment]
hello，after I updated my conda，the spyder is totally crashed。
Then I installed the lateset version 5.3.1 using "-forge", but an ERROR pumps up!
I have tried conda/pip uninstall and install the qtpy 2.1.0, but it dose NOT work at all! 
I'm seeking for help and thanks very much for your time~~~










At 2022-06-24 22:00:06, "Carlos Cordoba" ***@***.***> wrote:

Hey @guizi-code, thanks for reporting. This error is already fixed in our latest version (5.3.1).

Since it's still not available in Anaconda, you have two options:

Use our Windows installer, which you can find here.

Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:

 conda create -n spyder-cf -c conda-forge spyder
 conda activate spyder-cf
 spyder
